src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -309,6 +309,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
     ENDIF.
 
     ro_advanced_dropdown->add( iv_txt = 'Remove'
+                               iv_title = `Remove abapGit's records of the repository (SAP objects unaffected)`
```

Comment:
```suggestion
                               iv_title = `Remove abapGit's records of the repository (system objects will be unaffected)`
```

Reply:
I think "development object(s)" is the term used in other environments (SE80/ADT).

Reply:
Thank you! How is this?
```suggestion
                               iv_title = `Remove abapGit's records of the repository (the system's development objects will remain unaffected)`
```
![image](https://user-images.githubusercontent.com/34446306/99937206-c5e90500-2d6d-11eb-8524-060112379f9b.png)